client/src/components/Applications/AppLayout/AppLayout.test.js

```diff
@@ -19,13 +20,28 @@ function renderAppsHeader(store, appId) {
     <Provider store={store}>
       <MemoryRouter initialEntries={[`/applications/${appId}`]}>
         <Route path='/:appId?'>
-          <AppsHeader categoryDict={appTrayExpectedFixture}/>
+          <AppsLayout categoryDict={appTrayExpectedFixture}/>
```

Comment:
Was this change intended? `AppsLayout` does not have `categoryDict` property, and the rendering function is called `renderAppsHeader`. If you need to render another component, you can create a new rendering function like `renderAppsLayout`

Reply:
The change intended because I was import as app layout` import AppsLayout from "./AppLayout"` it will be work same as before but name is only thing just changed(since import as default and on AppForm.js on last line`export default AppsRoutes`;. I didn't need to render another component other than App header so I wouldn't need to.

Reply:
@williamkwon97 Even though the test passes, the syntax is not quite right, as `AppsLayout` has no `categoryDict` prop. You should preserve the `AppsHeader` code here, and add more code to handle your `AppsLayout` changes